core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-shell/dot-experiments-shell.module.ts

```diff
@@ -0,0 +1,38 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { RouterModule, Routes } from '@angular/router';
+import { DotExperimentsListModule } from '../dot-experiments-list/dot-experiments-list.module';
+import { DotExperimentsListComponent } from '../dot-experiments-list/dot-experiments-list.component';
+import { DotExperimentsShellComponent } from './dot-experiments-shell.component';
+import { DotExperimentsStore } from '../shared/stores/dot-experiments-store.service';
+import { DotLoadingIndicatorModule } from '@components/_common/iframe/dot-loading-indicator/dot-loading-indicator.module';
+
+const routes: Routes = [
+    { path: '', redirectTo: 'list', pathMatch: 'full' },
+    {
+        path: '',
+        component: DotExperimentsShellComponent,
+        children: [
+            {
+                path: 'list',
+                component: DotExperimentsListComponent
```

Comment:
You have a routing module for this, I thought it was for lazy loading but is not here.

Reply:
Fixed.